### PR TITLE
SNEED VENDORS WHOO!!!!!

### PR DESCRIPTION
**Pull request by Aphast**
## About The Pull Request
Adds a point vendor seed vendor for hydro to drain the service budget on weed

## Why It's Good For The Game
more good boy points shit and general QOL

~~also adds an extra 'n' to megaseed vendors~~

## Changelog
🆑 
add:Megasneed vendors
/:cl:

### CODE REVIEW

**Comment by TheChosenEvilOne** [comment]
your changelog is a bit fucked (missing the starting :cl:), also what exactly do you need help with?

**Comment by Aphast** [comment]
> 

i dont know why it does that, it happens with alot of my pr's, but what i need help with is when trying to spawn said vendor and entering in point_vendor all i get is pv/sec,eng,med,drug but not the seed one, like its not being read by something or idk

**Comment by Aphast** [comment]
in dream maker yes the box is checkmarked

**Comment by Aphast** [comment]
image
when attempting to spawn a point vendor this is the only result

**Comment by karmaisblackandbluepilled** [comment]
how odd, that's acting like it isn't ticked. Did you compile after ticking it?
also, you need to use VSC to work with tg-based code
they're animals and replaced DM's compilation with some frankensteinian shit

**Comment by Aphast** [comment]
forgive my lack of knowlge but whats VSC?

**Comment by Aphast** [comment]
did another full compile and now its in, doing map edits now

**Comment by karmaisblackandbluepilled** [comment]
vsc is visual studio code

**Comment by Aphast** [comment]
map stuff done, all should be good now

File: code/modules/vending/megaseed.dm [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /obj/machinery/vending/hydroseeds
-	name = "\improper MegaSeed Servitor"
+	name = "\improper MegaSneed Servitor"
```
Comment (TheChosenEvilOne) [inline code/modules/vending/megaseed.dm:2]: just keep this as it was

**Comment by TheChosenEvilOne** [comment]
sneed

**Comment by EtraIV** [comment]
you forgot to update icebox

**Comment by EtraIV** [comment]
also we probably should have testmerged this

**Comment by NullFag** [comment]
> forgive my lack of knowlge but whats VSC?

The smartest MerchantStation coder.

**Pull request merged**